packages/aws-cdk-lib/aws-s3-deployment/README.md

```diff
@@ -362,7 +362,7 @@ new s3deploy.BucketDeployment(this, 'DeployWithSignedPayloads', {
 
 ## Size Limits
 
-The default memory limit for the deployment resource is 128MiB. If you need to
+The default memory limit for the deployment resource is 512MiB when the feature flag `@aws-cdk/aws-s3-deployment:defaultMemoryLimit` is enabled, otherwise 128MiB. If you need to
```

Comment:
```suggestion
The default memory limit for the deployment resource is 512MiB. If you need to
```

Let's make it this way as the FF default value is true. We don't need to mention FF in the README.